src/models/image-map.ts

```diff
@@ -417,16 +446,69 @@ export const localAssetsImagesHandler: IImageHandler = {
     return url ? url.startsWith("assets/") || url.startsWith("curriculum/") : false;
```

Comment:
I think this will have to be updated assuming the URLs in the new content will no longer start with "curriculum". 
Perhaps the best match is to check that they don't have a `:` so that way we know they are relative.